package.json

```diff
@@ -39,23 +41,16 @@
     "build:plugin:webp-uploads": "webpack --mode production --env plugin=webp-uploads",
     "format-js": "wp-scripts format",
     "lint-js": "wp-scripts lint-js",
-    "format-php": "composer format",
     "phpstan": "composer phpstan",
-    "lint-php": "composer lint:all",
+    "lint:php": "composer lint",
+    "lint:php:fix": "composer format",
+    "lint:php:plugins": "cross-env WPP_PLUGIN=$npm_config_plugin php -f bin/phpcs/lint.php",
+    "lint:php:plugins:fix": "cross-env WPP_FIX=1 WPP_PLUGIN=$npm_config_plugin php -f bin/phpcs/lint.php",
```

Comment:
Nit-pick: The other `package.json` commands use hyphens (`-`), not colons (`:`), so better to stick with that for consistency. Also I would suggest we stick with the "format" name instead of "lint...fix", which is more intuitive to remember and aligned with the similar JS tooling.

Reply:
I think the build commands just change that and even in composer we are using `:` notation and this is pretty much common even in the Gutenberg repo, and the same with the `:fix` notation.